Cargo.toml

```diff
@@ -117,7 +117,7 @@ libp2p-gossipsub = { version = "0.37.0", path = "protocols/gossipsub", optional
 async-std = { version = "1.6.2", features = ["attributes"] }
 async-trait = "0.1"
 env_logger = "0.9.0"
-clap = {version = "3.1.6", features = ["derive"]}
+structopt = "0.3.21"
```

Comment:
Accidental merge, or is it on purpose to revert https://github.com/libp2p/rust-libp2p/pull/2600?

Reply:
accidental, will remove